### PR TITLE
Note Cortex is a CNCF project; make commercial references more balanced

### PR DESCRIPTION
**Pull request by bboreham**
Now more balanced re commercial services.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,18 +68,16 @@ To update dependencies, run
 To learn more about Cortex, consult the following documents / talks:
 
 - [Original design document for Project Frankenstein](http://goo.gl/prdUYV)
-- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
-- KubeCon Prometheus Day talk "Weave Cortex: Multi-tenant, horizontally scalable Prometheus as a Service" [slides](http://www.slideshare.net/weaveworks/weave-cortex-multitenant-horizontally-scalable-prometheus-as-a-service)
+- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/cortexproject/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
+- KubeCon Prometheus Day talk "Weave Cortex: Multi-tenant, horizontally scalable Prometheus as a Service" [slides](http://www.slideshare.net/cortexproject/weave-cortex-multitenant-horizontally-scalable-prometheus-as-a-service)
```
Comment (tomwilkie) [inline README.md]: These links should probably stay pointing at Weaveworks?

File: README.md [hunk 2]
```diff
@@ -69,18 +68,16 @@ To update dependencies, run
...
 
 If you have any questions regarding Cortex, or on using Prometheus with Weave Cloud:
 
 
-- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> Slack channel. To invite yourself to the Weave Community Slack channel, visit https://weaveworks.github.io/community-slack/.
-- Join the <a href="https://www.meetup.com/pro/Weave/"> Weave User Group </a> and get invited to online talks, hands-on training and meetups in your area.
-- Send an email to <a href="mailto:weave-users@weave.works">weave-users@weave.works</a>
-- <a href="https://github.com/weaveworks/cortex/issues/new">File an issue.</a>
+- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> Slack channel. To invite yourself to the Weave Community Slack channel, visit https://cortexproject.github.io/community-slack/.
+- <a href="https://github.com/cortexproject/cortex/issues/new">File an issue.</a>
```
Comment (tomwilkie) [inline README.md]: Maybe add a link to https://groups.google.com/forum/#!forum/cortex-monitoring?

**Comment by tomwilkie** [comment]
Would you mind if I stuck some blurb in for Grafana Cloud on this branch?

File: README.md [hunk 3]
```diff
@@ -69,18 +68,16 @@ To update dependencies, run
...
 ## <a name="help"></a>Getting Help
 
 If you have any questions regarding Cortex, or on using Prometheus with Weave Cloud:
 
 
-- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> Slack channel. To invite yourself to the Weave Community Slack channel, visit https://weaveworks.github.io/community-slack/.
-- Join the <a href="https://www.meetup.com/pro/Weave/"> Weave User Group </a> and get invited to online talks, hands-on training and meetups in your area.
-- Send an email to <a href="mailto:weave-users@weave.works">weave-users@weave.works</a>
-- <a href="https://github.com/weaveworks/cortex/issues/new">File an issue.</a>
+- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> Slack channel. To invite yourself to the Weave Community Slack channel, visit https://cortexproject.github.io/community-slack/.
```
Comment (dholbach) [inline README.md]: https://slack.cncf.io/ for the invite, https://cloud-native.slack.com/messages/cortex for the channel.

File: README.md (hunk 2 above)
Comment (dholbach) [reply README.md]: Chris brought up cortex-users@lists.cncf.io - not sure if that's already created.

File: README.md (hunk 1 above)
Comment (bboreham) [reply README.md]: fixed

**Comment by bboreham** [comment]
@tom please do add blurb; I think mostly each service should have a link to detailed info elsewhere.
Should probably be phrased as "sponsored by" or similar.

File: README.md [hunk 4]
```diff
@@ -75,12 +74,11 @@ To learn more about Cortex, consult the following documents / talks:
...
+If you have any questions regarding Cortex:
 
 
-- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> Slack channel. To invite yourself to the Weave Community Slack channel, visit https://weaveworks.github.io/community-slack/.
-- Join the <a href="https://www.meetup.com/pro/Weave/"> Weave User Group </a> and get invited to online talks, hands-on training and meetups in your area.
-- Send an email to <a href="mailto:weave-users@weave.works">weave-users@weave.works</a>
-- <a href="https://github.com/weaveworks/cortex/issues/new">File an issue.</a>
+- Ask a question on the [Cortex Slack channel](https://cloud-native.slack.com/messages/cortex/). To invite yourself to the CNCF Slack, visit http://slack.cncf.io/.
+- <a href="https://github.com/cortexproject/cortex/issues/new">File an issue.</a>
+Send an email to <a href="mailto:cortex-users@lists.cncf.io">cortex-users@lists.cncf.io</a>
```
Comment (dholbach) [inline README.md]: It's not listed here yet: https://lists.cncf.io/g/main/subgroups

Comment (bboreham) [reply README.md]: Chris said the request had been made.  Nobody uses email these days anyway ;-)

Comment (dholbach) [reply README.md]: :sadpanda:

**Comment by cezzaine** [comment]
Please update the logo in the README to this one...
cortex_logo

**Pull request merged**